okhttp/src/main/java/okhttp3/internal/platform/Platform.kt

```diff
@@ -80,27 +77,17 @@ open class Platform {
   /** Prefix used on custom headers.  */
   fun getPrefix() = "OkHttp"
 
-  open fun getSSLContext(): SSLContext = try {
-    SSLContext.getInstance("TLS")
-  } catch (e: NoSuchAlgorithmException) {
-    throw IllegalStateException("No TLS provider", e)
-  }
+  open fun newSSLContext(): SSLContext = SSLContext.getInstance("TLS")
```

Comment:
this seemed misnamed, especially since we mutate it immediately after access

Reply:
Good call